server/app/services/validateFormData.js

```diff
@@ -0,0 +1,17 @@
+const Joi = require("joi");
```

Comment:
password and email must be validated with pattern (regex).
GPDR Rules actually requires atleast (8 characters, 1 uppercase, 1 lowercase, a number, and a special character). In near future, 16 characters... 😱

Reply:
Done !